config/private.example.yml

```diff
@@ -10,7 +10,7 @@ test:
 
 development:
   <<: *default
-  api_path: https://cdrhdev1.unl.edu/api/v1/collection/chesnutt/
+  api_path: http://localhost:3000/collection/habeascorpus
```

Comment:
Don't think we want the collection name in the example to be habeascorpus :wink:

Please change the hostname part of the URLS to just be "https://hostname.unl.edu" like we've done with other private example files.